packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -284,6 +303,36 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
     }
   }
 
+  /// Returns the intent, if any, that matches the current set of pressed keys.
+  Intent? matchingIntent({
+    KeySet<KeyboardKey>? keySet,
+  }) {
+    if (keySet == null && RawKeyboard.instance.keysPressed.isEmpty) {
+      return null;
+    }
+    keySet ??= LogicalKeySet.fromSet(RawKeyboard.instance.keysPressed);
+    Intent? matchedIntent = _shortcuts[keySet];
+    if (matchedIntent == null) {
+      // If there's not a more specific match, We also look for any keys that
```

Comment:
Don't we already have to do this matching to dispatch the actual key-press? Can we re-use that logic here?

Reply:
Never mind, just scrolled down and saw that the re-use is the other way around. That works :)